db/migrate/20220916163432_add_status_to_merchants.rb

```diff
@@ -1,5 +1,5 @@
 class AddStatusToMerchants < ActiveRecord::Migration[5.2]
   def change
-    add_column :merchants, :status, :integer, default: 0
+    add_column :merchants, :status, :integer, default: 1
```

Comment:
I had an idea that could work, in case anyone runs into a similar issue in the future, if you wanted to avoid updating a migration, you could swap the numbers in the enum status in the model instead